Lib/typing.py

```diff
@@ -325,8 +325,8 @@ def inner(*args, **kwds):
 def _eval_type(t, globalns, localns, recursive_guard=frozenset()):
     """Evaluate all forward references in the given type t.
     For use of globalns and localns see the docstring for get_type_hints().
-    recursive_guard is used to prevent prevent infinite recursion
-    with recursive ForwardRef.
+    recursive_guard is used to prevent infinite recursion with a recursive
+    ForwardRef.
```

Comment:
I found this while researching bpo-46581. This edit removes a duplicate "prevent" and re-justifies text. I'm happy to remove this edit if it was intended as a subtle recursion prevention joke that was lost on me.